### PR TITLE
fix(package): always use latest published version "TEST"

### PR DESCRIPTION
**Pull request by muhabdulkadir**
# Closes #<CORRESPONDING ISSUE NUMBER>

### Changes

- resolve concert intra repo dependencies to always use latest npm published package

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [ ] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [ ] Merging to `main` from `fork:branchname`

### CODE REVIEW

File: packages/concerto-core/package.json [hunk 1]
```diff
@@ -64,9 +64,9 @@
     "yargs": "17.7.2"
   },
   "dependencies": {
-    "@accordproject/concerto-cto": "file:../concerto-cto",
+    "@accordproject/concerto-cto": "*",
```
Comment (sanketshevkar) [inline packages/concerto-core/package.json:67]: Are we sure we want this? In the future when we release v4, and if some want to install `concerto-core@3.19.9`, won't this forcefully install `concerto-util@4.0.0` and `concerto-cto@4.0.0` which themselves could have a breaking change and thus breaking code for `concerto-core@3.19.9`?

Comment (muhabdulkadir) [reply packages/concerto-core/package.json:67]: This is a test.

File: lerna.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
```
Comment (sanketshevkar) [inline lerna.json:2]: We've moved to npm workspaces I believe.

**Review by sanketshevkar — CHANGES_REQUESTED** [review]

File: lerna.json (hunk 2 above)
Comment (sanketshevkar) [reply lerna.json:2]: We don't use lerna anymore.

Comment (sanketshevkar) [reply lerna.json:2]: https://github.com/accordproject/concerto/pull/912

**Pull request closed without merging**